src/server/app.js

```diff
@@ -124,6 +139,24 @@ app.use((req, res, next) => {
 		});
 	}
 
+	if (!searchConfig.node || !searchConfig.auth) {
+		let msg;
+		if (process.env.DEPLOY_ENV === 'test') {
+			msg = 'Elastic search configs are not provided. Search server is not available on the test website';
+		}
+		else if (process.env.DEPLOY_ENV === 'beta') {
+			msg = 'Elastic search configs are not provided. Search server is not available on the beta website';
+		}
+		else {
+			msg = 'Elastic search configs are not provided. Search server is not available in your local environment';
```

Comment:
I think we can do a single message for all environments, maybe something like 
'We could not connect to our search server, all search functionality is unavailable.'